src/smartbanner.scss

```diff
@@ -4,7 +4,7 @@ $blue: #1474fc;
 
 .smartbanner {
   position: absolute;
-  top: 0;
+  top: -84px;
```

Comment:
We can't make changes to existing styles, that will break it for the current users. 

Let's please keep all the current styles intact and make Android-related changes in `smartbanner--android`.